### PR TITLE
Added code to remove alert banner if alert data doesn't exist 

### LINKED ISSUE #1775: Alert: Alert banner needing a browser reload
A browser with the alert banner still shows the alert banner even after the expiration date unless refreshed.

To clarify, a browser that displayed the alert banner when the alert data was posted may sometimes continue to show the alert banner the next day if that browser tab remains open. However, any new tabs opened after the alert expiration date would not show the alert banner. This issue was reported by a person using the Chrome browser. The affected alert is for a specific channel that the Alert project listens to.

### PR DESCRIPTION
**Pull request by sarongebre**
Closes #1775 
Renamed variables and functions. Added comments.

I have been having some issues with the alert data URL not updating when the cache is enabled.

I have this in the code 
 // Are these two variables below needed?
        // $alertToggle = $('#unlalert_toggle'),
        // $alertContent = $('#_content'),

### CODE REVIEW

File: wdn/templates_5.3/js-src/unlalert.babel.js [hunk 1]
```diff
@@ -3,146 +3,162 @@ define([
...
+  activeAlertCookieName = 'unl_alerts_active_alert',
+  
+  serverCallExecutionDelay = 30000, // Delay (in milliseconds) before executing the checkIfServerCallIsNeeded function
+  serverCallExecutionDelayTwo = 31000, // Secondary delay (in milliseconds) before executing the checkIfServerCallIsNeeded function
+
+  alertDataRecievedCookieMaxAge = 30, // Maximum age (in seconds) for the alertDataReceived cookie
+  closedAlertCookieMaxAge = 3600, // Maximum age (in seconds) for the alertDataReceived cookie
+
+  // Sets a cookie to indicate unlAlerts.data has been recieved
+  _setAlertDataReceivedCoookie = function() {
```
Comment (tommyneu) [inline wdn/templates_5.3/js-src/unlalert.babel.js:21]: Coookie

File: wdn/templates_5.3/js-src/unlalert.babel.js [hunk 2]
```diff
@@ -3,146 +3,162 @@ define([
...
-  _getClosedAlerts = function() {
-    var c = WDN.getCookie(ckPrfx + 'C');
-    if (c) {
-      return c.split(',');
+  var alertDataUrl = 'https://alert.unl.edu/json/unlcap.js';
+  var alertDataActiveIDs = [], 
+  serverCallTimeoutID,
+  
+  closedAlertCookieName = 'unl_alerts_closed_alerts',
+  alertDataRecievedCookieName = 'unl_alerts_data_recieved',
```
Comment (tommyneu) [inline wdn/templates_5.3/js-src/unlalert.babel.js:11]: Recieved

File: wdn/templates_5.3/js-src/unlalert.babel.js [hunk 3]
```diff
@@ -3,146 +3,162 @@ define([
...
-
-  _getClosedAlerts = function() {
-    var c = WDN.getCookie(ckPrfx + 'C');
-    if (c) {
-      return c.split(',');
+  var alertDataUrl = 'https://alert.unl.edu/json/unlcap.js';
+  var alertDataActiveIDs = [], 
+  serverCallTimeoutID,
+  
+  closedAlertCookieName = 'unl_alerts_closed_alerts',
```
Comment (tommyneu) [inline wdn/templates_5.3/js-src/unlalert.babel.js:10]: Keep cookie names the same

File: wdn/templates_5.3/js-src/unlalert.babel.js (hunk 1 above)
Comment (sarongebre) [reply wdn/templates_5.3/js-src/unlalert.babel.js]: Changed it to be Cookie

File: wdn/templates_5.3/js-src/unlalert.babel.js (hunk 2 above)
Comment (sarongebre) [reply wdn/templates_5.3/js-src/unlalert.babel.js]: Changed it to be Received.

File: wdn/templates_5.3/js-src/unlalert.babel.js (hunk 3 above)
Comment (sarongebre) [reply wdn/templates_5.3/js-src/unlalert.babel.js]: Changed cookie names to their original values. 

closedAlertCookieName = 'unlAlertsC',
 alertDataReceivedCookieName = 'unlAlertsData',
 activeAlertCookieName = 'unlAlertsA',

**Review by tommyneu — APPROVED** [review]

**Pull request merged**